ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.directive.js

```diff
@@ -483,20 +483,23 @@ export function specEditorDirective($rootScope, $templateCache, $injector, $sani
             $state.go(graphicalState.name);
         };
 
-        specEditor.getParameterIssues = () => {
-            return scope.model.issues
-                .filter((issue) => (issue.group === 'parameters'))
-                .concat(Object.values(scope.model.getClusterMemberspecEntities())
-                    .filter((spec) => (spec && spec.hasIssues()))
-                    .reduce((acc, spec) => (acc.concat(spec.issues)), []));
-        };
-        scope.getConfigIssues = specEditor.getConfigIssues = () => {
+        /**
+         * Gets collection of issues filtered by group.
+         *
+         * @param {String} groupName The group name to filter issues by.
+         * @returns {[]} The collection of issues found.
+         */
+        scope.getIssuesByGroup = (groupName) => {
```

Comment:
This is just a refactoring, now this method can be used to get issues for any relevant group.